### ISSUE TITLE
ImportError on protobuf

### ISSUE THREAD

**Issue by toslunar** [chainer/chainer-test#481]
Daily tests of Chainer with some config are failing.
```
tests/chainer_tests/links_tests/caffe_tests/test_caffe_function.py:11: in <module>
    from chainer.links import caffe
chainer/links/caffe/__init__.py:1: in <module>
    from chainer.links.caffe.caffe_function import CaffeFunction  # NOQA
chainer/links/caffe/caffe_function.py:10: in <module>
    from chainer.links.caffe.protobuf3 import caffe_pb2 as caffe_pb
chainer/links/caffe/protobuf3/caffe_pb2.py:7: in <module>
    from google.protobuf import descriptor as _descriptor
/opt/pyenv/versions/3.6.6/lib/python3.6/site-packages/protobuf-3.3.0-py3.6-linux-x86_64.egg/google/protobuf/descriptor.py:46: in <module>
    from google.protobuf.pyext import _message
E   ImportError: /opt/pyenv/versions/3.6.6/lib/python3.6/site-packages/protobuf-3.3.0-py3.6-linux-x86_64.egg/google/protobuf/pyext/_message.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZNK6google8protobuf10TextFormat17FieldValuePrinter9PrintBoolEb
```

**Comment by toslunar** [comment]
#485 probably closes this.

**Comment by kmaehashi** [comment]
Fixed via #485
